controllers/update_status.go

```diff
@@ -586,6 +599,9 @@ func validateProcessGroup(ctx context.Context, r *FoundationDBClusterReconciler,
 	if !incorrectPVC && desiredPvc != nil {
 		incorrectPVC = !metadataMatches(currentPVC.ObjectMeta, desiredPvc.ObjectMeta)
 	}
+	if incorrectPVC {
```

Comment:
What is the benefit for logging this change but not the others? Should that be rather logged inside the `UpdateCondition` method?

Reply:
It makes it much easier to figure out why validateProcessGroup() doesn't behave as I would expect. this happens when I added unit test for validateProcessGroup() and it fails due to incorrectPVC parameters.

Reply:
Even in prod env, it helps to diagnose why PVC is incorrect, IMO.

Reply:
Not sure about this. But I'm okay with keeping the log line if you think it's useful.